SQL/New_patches/2019-12-17-RemovePasswordExpiry.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE users DROP COLUMN Password_expiry;
+ALTER TABLE users ADD COLUMN Password_expired TINYINT(1) NOT NULL DEFAULT 0 AFTER Password_hash;
```

Comment:
Add the new column,
UPDATE users SET Password_expired = 0 where Password_expiry > timestamp;
UPDATE users SET Password_expired = 1 where Password_expired IS NULL;
drop old column

Setting a default value will not change existing rows, only set the default for new entries.

Reply:
@johnsaigle You also have to specify 0 where still valid. Adding a new column fill it with `null` value.

Reply:
Won't DEFAULT 0 take care of that?

Reply:
Yes, sorry, I was still tinking past version or maybe a mix with Oracle (again older version).